sample/lit-todo/main.js

```diff
@@ -0,0 +1,193 @@
+import {handleDragStart, isDragging} from './drag.js';
+import Replicache, {REPMWasmInvoker} from 'replicache/out/mod.js';
+import {html, render} from 'lit-html';
+import {classMap} from 'lit-html/directives/class-map.js';
+import {live} from 'lit-html/directives/live';
+import {repeat} from 'lit-html/directives/repeat.js';
+import '@material/mwc-checkbox/mwc-checkbox.js';
+import '@material/mwc-fab';
+import '@material/mwc-icon';
+import '@material/mwc-icon-button';
+import '@material/mwc-textfield';
+import '@material/mwc-top-app-bar/mwc-top-app-bar.js';
+
+const key = id => `/todo/${id}`;
+
+let rep = new Replicache({
+  // URL of the diff server to use. The diff server periodically fetches
+  // the 'client view' from your service and returns any delta. You can
+  // use our hosted diff server (as here) or a local diff server, which
+  // is useful during development. See
+  // https://github.com/rocicorp/replicache#server-side for more
+  // information.
+  diffServerURL: 'https://serve.replicache.dev/pull',
+
+  // Auth token for the diff server, if any.
+  diffServerAuth: '1',
+
+  // URL of your service's Replicache batch endpoint. Replicache
+  // will send batches of mutations here for application when the
+  // network is available.
+  batchURL: 'https://replicache-sample-todo.now.sh/serve/replicache-batch',
+
+  // Auth token for your client view and batch endpoints, if any.
+  dataLayerAuth: '2',
+  pushDelay: 300,
+
+  repmInvoker: new REPMWasmInvoker(
+    './output/wasm/release/replicache_client_bg.wasm',
```

Comment:
Should probably use import.meta.url but whatevs

Reply:
Actually, do you really need this? Doesn't the default work with rollup?

Reply:
No, the wasm bundle really needs to be a separate resource from the big bundle, I think. And without this, the script doesn't know where to find the wasm file.

Reply:
The wasm file is relative to `repm-invoker.js` so as long as we keep the file structure that should still work. With rollup we end up with a singe js file (I think) but we should probably put the wasm files as direct siblings to repm-invoker and just use `./replicache_client.js`

Reply:
The problem is that lower-level code currently defaults the name of the wasm module to name of including js module, which in this case in 'main.js', so it looks for 'main.wasm'.

Reply:
Yes. `replicache_client.js` uses its current path and appends `_bg.wasm` and with rollup `replicache_client.js` gets folded into the main output bundle.